src/boundary/Obstacle.cpp

```diff
@@ -210,18 +194,18 @@ void Obstacle::createObstacle(size_t Nx, size_t Ny) {
         }
     }
 
-    //// INNER of OBSTACLE
-    //// fill oInner list with inner indices of obstacles
-    //for (size_t k = 1; k < strideZ - 1; ++k) {
-    //    for (size_t j = 1; j < strideY - 1; ++j) {
-    //        for (size_t i = 1; i < strideX - 1; ++i) {
-    //            size_t index = (i - 1) + (strideX - 2) * (j - 1) + (strideX - 2) * (strideY - 2) * (k - 1);
+    // // INNER of OBSTACLE
+    // // fill oInner list with inner indices of obstacles
+    // for (size_t k = 1; k < strideZ - 1; ++k) {
+    //     for (size_t j = 1; j < strideY - 1; ++j) {
+    //         for (size_t i = 1; i < strideX - 1; ++i) {
+    //             size_t index = (i - 1) + (strideX - 2) * (j - 1) + (strideX - 2) * (strideY - 2) * (k - 1);
 
-    //            size_t idx = IX(i, j, k, strideX, strideY);
-    //            *(m_obstacleInner + index) = m_obstacleList[idx];
-    //        }
-    //    }
-    //}
+    //             size_t idx = IX(i, j, k, strideX, strideY);
+    //             *(m_obstacleInner + index) = m_obstacle_list[idx];
+    //         }
+    //     }
+    // }
```

Comment:
is this code not longer needed -> delete

Reply:
5c97868f4303b2d83989a6c476cfcb68930c7c12